pastis/launchers/run_luvoir_efields.py

```diff
@@ -24,10 +24,12 @@
         fpath = CONFIG_PASTIS.get('LUVOIR', 'harris_data_path')  # path to Harris spreadsheet
         pad_orientations = np.pi / 2 * np.ones(CONFIG_PASTIS.getint('LUVOIR', 'nb_subapertures'))
         DM_SPEC = (fpath, pad_orientations, True, False, False)
+        NUM_MODES = 5  # TODO: works only for thermal modes currently
 
     # If using Segmented Zernike Mirror
-    if WHICH_DM == 'seg_mirror':
+    if WHICH_DM in ['seg_mirror', 'zernike_mirror']:
         DM_SPEC = 3
+        NUM_MODES = DM_SPEC
```

Comment:
I guess you added 'zernike_mirror' to do analysis for global zernike modes?

Reply:
Correct. I don't think it would work if we were to just run it blindly, but at least we remain the idea that this *should* be made possible further down the line.